src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -253,6 +260,16 @@
     public static $manyMethods = ['belongsToMany', 'morphToMany', 'morphedByMany'];
 
     /**
+     * All of the valid primitive cast types.
+     *
+     * @var array
+     */
+    protected static $primitiveCastTypes = [
```

Comment:
No support in HHVM upto 3.12.0:
**Warning: Constants may only evaluate to scalar values**